addon/components/connect.js

```diff
@@ -37,9 +38,16 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
           });
         }
 
-        this.actions = Object.assign({},
-          this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
-        );
+        if (typeof dispatchToActions === 'function') {
+          this.actions = Object.assign({},
+            this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
+          );
+        }
+
+        if (typeof dispatchToActions === 'object') {
+          this.actions = Object.assign({},
+            this.actions, bindActionCreators.call(this, dispatchToActions, redux.dispatch.bind(redux)));
```

Comment:
Given that this doesn't appear to bind `this`, could you just change this to:
```
bindActionCreators(dispatchToActions, redux.dispatch.bind(redux));
```

That way it's not misleading to someone who might be reading this in the future.